web-local/src/lib/components/column-profile/column-types/sparks/NullPercentageSpark.svelte

```diff
@@ -9,6 +9,7 @@
   export let type: string;
   export let nullCount: number;
   export let totalRows: number;
+  export let isFetching: boolean;
 
   $: percentage = nullCount / totalRows;
```

Comment:
I think this is _almost_ right; the issue here, though, is that we should only update the number if both the nullCount & totalRows are both updating (in other words, `isFetching === false`). I would maybe opt to change this to 
```suggestion
  $: if(!isFetching) percentage = nullCount / totalRows;
```

And get rid of the templated check with the `..%`. Otherwise this is actually a design tweak. The default value of `percentage` should be `undefined` in that case, should not show anything in the template.

Reply:
Made the change. Also added a check for `percentage <=1` as there were cases when percentage momentarily became >100% when editing the model quickly.

Reply:
Thanks! I'll take a look now.